Services/Interfaces/IApiRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+using Interfaces.Models;
+
+namespace Interfaces
+{
+    public interface IApiRequest
```

Comment:
For me that name looks confusing. Could you please rename it to the f.e. ILastFmService